api/docs/v2/storefront/index.yaml

```diff
@@ -5443,6 +5429,14 @@ components:
       schema:
         type: string
       example: '1'
+    CardId:
+      name: card_id
```

Comment:
@MatthewKennedy this is `id` in the URL structure

Reply:
Hi @damianlegawiec,

The variable would look like this in the docs,:
<img width="990" alt="Screenshot 2021-08-11 at 11 28 36" src="https://user-images.githubusercontent.com/1240766/129014286-22bc4331-11fe-4aee-8626-e264cd0dc579.png">

Reply:
The request gets formulated okay

Reply:
I guess `Id` isn't so bad, as it is always just a number, but we couldn't do the same with `Slug`, because the examples would fail, a product slug in the example isn't going to pull up a page and vice-vera.

On this note, we currently have `IdOrPermalink` in use for both `Taxon` and `Product` for the in path request params.

The example shows a `"knitted-high-neck-sweater"` this won't work for a Taxon, but will fetch a product.

Going further, the param is named IdOrPermalink and a product has a `slug` not a `permalink`, which thinking over this more as I write, I'm thinking it would be better to have the dedicated variables for each request, Vs trying to share them.